src/templates/games/game_grid.html

```diff
@@ -54,7 +54,7 @@ <h1 class="display-1 mt-3 mb-3">Games</h1>
             </div>
             <div class="card-body d-flex flex-column">
               <h4 class="card-title">{{ game.name }}</h4>
-              <p class="card-text flex-grow-1">{{ game.description|linebreaksbr }}</p>
+              <p class="card-text flex-grow-1">{{ game.description|linebreaksbr|truncatewords_html:20 }}</p>
```

Comment:
<img width="385" alt="image" src="https://github.com/uchicago-cs/chigame/assets/81624602/d8596b8e-844b-4cad-9a79-1a3668eb5132">

The description in the game grid looks perfect! I was thinking, is it possible to let the user know/sth that can signal to them that they can read more about the game description so they can know the game description was purposely truncated and there's a complete version of it? Can a "read more" under the description be a useful indicator or would it be redundant since the whole grid itself is a link to the game detail page?

Reply:
Similar to what I said in the comment above, I think this is a really good suggestion, but I'd rather leave it up to future team members to open up a new game card styling issue. The main focus of this issue was to make sure that descriptions don't make the cards too big.

Reply:
Like with the previous comment, I think your judgement makes sense. I can resolve these comments now